tslint.json

```diff
@@ -27,6 +30,7 @@
     "arrow-parens": [
       true,
       "ban-single-arg-parens"
-    ]
+    ],
+    "no-console": false
```

Comment:
Did we have no-console: false before? Because there should be no console.logs anywhere in the `src`. If we need it for testing, we should have special rules for specific folders

Reply:
The only directory that `tslint.json` lints right now is the `__tests__` directory, so yeah it's in the tests.